resources/metadata/vaultRotateSecretId.yaml

```diff
@@ -139,3 +140,54 @@ spec:
           - STAGES
           - STEPS
         description: The Azure DevOps pipeline ID. Also called as definition ID
+      - name: githubToken
+        aliases:
+          - name: access_token
```

Comment:
Hyperspace adds this to Vault with the name `access_token` but I would also add the name `token` as alias as for the other orchestrators.

Reply:
Done